src/Sulu/Component/Persistence/Repository/ORM/OrderByTrait.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/*
+ * This file is part of the Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\Persistence\Repository\ORM;
```

Comment:
@chirimoya mentioned we should create an own namespace `Sulu\Component\Persistence\Repository\ORM\Behavior` for that